tree/ntuple/v7/src/RField.cxx

```diff
@@ -895,6 +919,27 @@ ROOT::Experimental::RClassField::RClassField(std::string_view fieldName, std::st
       Attach(std::move(subField),
 	     RSubFieldInfo{kDataMember, static_cast<std::size_t>(dataMember->GetOffset())});
    }
+
+   // Add post-read callbacks for I/O customization rules; only rules that target transient members are allowed for now
+   // TODO(jalopezg): revise after supporting schema evolution
+   if (const auto ruleset = fClass->GetSchemaRules()) {
+      auto referencesNonTransientMembers = [this](const ROOT::TSchemaRule *rule) {
+         R__ASSERT(rule->GetTarget() != nullptr);
+         for (auto target : ROOT::Detail::TRangeStaticCast<TObjString>(*rule->GetTarget())) {
+            const auto dataMember = fClass->GetDataMember(target->GetString());
+            if (dataMember && dataMember->IsPersistent()) {
```

Comment:
What if `dataMember == nullptr`?

Reply:
That means there's at least a member mentioned in the `target` list that is not part of the current definition of the class.
Hmmm...  I guess we should also fail in that case (?)